### PR TITLE
Switches

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

File: src/switch/switch.svelte [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  let _class = null;
+  export { _class as class };
+  export let inputClass = null;
+  export let trackClass = null;
+  export let thumbClass = null;
+
+  export let value = false;
+  export let disabled = false;
+  export let slotLeft = false;
```
Comment (aabounegm) [inline src/switch/switch.svelte:13]: I feel like the slot will almost always be used just for a label, so I feel like the name `labelLeft` would be more appropriate.

File: src/switch/switch.svelte [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<label class={classes('switch', _class)}>
+  {#if slotLeft}
+    <slot />
+  {/if}
+  <input
+    bind:checked={value}
+    type="checkbox"
+    class={inputClass}
+    {disabled}
+    on:change={(e) => dispatch('change', { value: e.target.checked })}
```
Comment (aabounegm) [inline src/switch/switch.svelte:27]: Same comment as in #14 about dispatched events
```suggestion
    on:change
```

File: src/switch/switch.svelte (hunk 1 above)
Comment (illright) [reply src/switch/switch.svelte:13]: This is kind of for consistency with Checkbox and RadioButton, which also have a `slotLeft` prop, while CheckboxGroup has `labelsLeft`. In that way, the Checkbox and RadioButton components are meant to be used sparingly, where extra customization is needed, and in that case the `slotLeft` name wouldn't scare people away. As for the switch, again, consistency.

File: src/switch/switch.svelte (hunk 2 above)
Comment (illright) [reply src/switch/switch.svelte:27]: Same thing, to be fixed later

**Pull request merged**